tests/functional/aws-node-sdk/test/bucket/put.js

```diff
@@ -36,6 +47,46 @@ describe('PUT Bucket - AWS.S3.createBucket', () => {
             bucketUtil = new BucketUtility('default', sigCfg);
         });
 
+        // Why describeSkipIfOldConfig?
+        // AWS returns 404 - NoSuchUpload in us-east-1. This behavior
+        // can be toggled to be compatible with AWS by enabling
+        // usEastBehavior in the config.
+        // describeSkipIfOldConfig
+        describeSkipIfOldConfig('create bucket twice', () => {
+            beforeEach(done => bucketUtil.s3.createBucket({ Bucket:
+              bucketName }, done));
+            afterEach(done => bucketUtil.s3.deleteBucket({ Bucket: bucketName },
+              done));
+            // AWS JS SDK send a request with locationConstraint us-east-1 if no
+            // locationConstraint provided.
+            it('should allow to recreate if no locationConstraints provided',
```

Comment:
should return a 200 if no locationConstraints provided.

(it doesn't actually recreate it just returns a success code instead of error)